### PR TITLE
Ghg feedback april/feat/316 several ameliorations

### PR DESCRIPTION
**Pull request by guilbep**
# Describe your changes

Several changes concerning bugs and bad behaviors

# Related GitHub issues and pull requests

- Ref: #316 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] Don't forget to link PR to issue if you are solving one.

### CODE REVIEW

File: frontend/src/components/green_house_gaz/generic/InstanceTable.vue [hunk 1]
```diff
@@ -256,6 +256,9 @@ export default class BaselineTable extends Vue {
       if (newDuplicate.input?.name) {
         newDuplicate.input.name = `${newDuplicate.input.name}_copy`;
       }
+      // to simulate new element so it can be inserted in endline
+      newDuplicate.increment = (this.maxItem ?? -1) + 1;
```
Comment (blueur) [inline frontend/src/components/green_house_gaz/generic/InstanceTable.vue:260]: -1 +1 ?

File: frontend/package.json [hunk 2]
```diff
@@ -36,6 +36,8 @@
     "vue-property-decorator": "^9.1.2",
     "vue-router": "^3.2.0",
     "vue2-leaflet": "^2.7.1",
+    "vuelidate": "^0.7.7",
```
Comment (blueur) [inline frontend/package.json:39]: why not https://v2.vuejs.org/v2/cookbook/form-validation.html ?

**Review by blueur — APPROVED** [review]

File: frontend/src/components/green_house_gaz/generic/InstanceTable.vue (hunk 1 above)
Comment (guilbep) [reply frontend/src/components/green_house_gaz/generic/InstanceTable.vue:260]: @blueur c'est ?? -1 :)
c'est une façon d'avoir 0 par défaut

File: frontend/package.json (hunk 2 above)
Comment (guilbep) [reply frontend/package.json:39]: @blueur I needed to have the $dirty logic. And I'm familiar with vuelidate.

**Pull request merged**